engine/config/1.0/schema.json

```diff
@@ -65,7 +65,7 @@
           "description": "The authentication method to use for the OAuth request.",
           "enum": ["", "client_secret_basic", "bearer_access_token"]
         },
-        "parameters": {
+        "params": {
```

Comment:
Typo fix. The Engine always used `params` here, I missed it when I wrote the initial schema.